src/explorer/SubscriptionTreeItem.ts

```diff
@@ -101,22 +100,22 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
 
         context.showCreatingTreeItem(nonNullProp(wizardContext, 'newSiteName'));
 
-        if (!advancedCreation) {
+        if (!context.advancedCreation) {
             await setDefaultRgAndPlanName(wizardContext, siteStep);
         }
 
         try {
             await wizard.execute();
         } catch (error) {
-            // if there is an error when creating, prompt the user to try advanced creation
-            // tslint:disable-next-line: strict-boolean-expressions
-            if (!parseError(error).isUserCancelledError && !advancedCreation) {
-                const message: string = `Modify the setting "${extensionPrefix}.${configurationSettings.advancedCreation}" if you want to change the default values when creating a Web App in Azure.`;
+            if (!parseError(error).isUserCancelledError && !context.advancedCreation) {
 
+                const message: string = `Use advanced create to change the default values when creating a Web App in Azure.`;
+                const btn: MessageItem = { title: 'Create Web App... (Advanced)' };
                 // tslint:disable-next-line: no-floating-promises
-                ext.ui.showWarningMessage(message, AppServiceDialogResponses.turnOnAdvancedCreation).then(async result => {
-                    if (result === AppServiceDialogResponses.turnOnAdvancedCreation) {
-                        await workspaceConfig.update('advancedCreation', true, ConfigurationTarget.Global);
+                ext.ui.showWarningMessage(message, btn).then(async result => {
```

Comment:
> I reset the context except for siteName and runtime so that the user does not have to re-enter their details.

Not really sold on this logic. I would expect a "fresh" create to start and I would re-enter the app name, etc. For example, I wonder if there's a situation where the wizard partially creates the site, it fails, the user clicks "Create Web App... (Advanced)" and then it fails again because you re-used the name of a partially failed site. Users can handle re-entering that info

Reply:
I'll create an issue and ask @fiveisprime to weigh in on it when he returns.